Subject: [PATCH] Work around bool type redefinition by altivec

On powerpc64le, the SDL header from SDL2 may include altivec.h and this
(combined with -std=c99) will cause a compile failure due to bool being
redefined as a vector type.

Re-include stdbool.h after SDL.h to restore the normal bool type.

Signed-off-by: Sam Bobroff <sam.bobroff@au1.ibm.com>
---
 audio/out/ao_sdl.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/audio/out/ao_sdl.c b/audio/out/ao_sdl.c
index d9d0062..0cb80d6 100644
--- a/audio/out/ao_sdl.c
+++ b/audio/out/ao_sdl.c
@@ -29,6 +29,7 @@
 #include "osdep/timer.h"
 
 #include <SDL.h>
+#include <stdbool.h>
 
 struct priv
 {
-- 
2.10.0.297.gf6727b0

